services/fission-user-workers/service/dockerworker/main.py

```diff
@@ -0,0 +1,98 @@
+import requests
+import os
+import sys
+import importlib
+from zipfile import ZipFile
+from base64 import b64decode
+import pika
+import flask
+import time
+
+app = flask.Flask("dockerworker")
+
+
+def main():
+    print("Worker starting...\nValidating environt values...")
+    # Load all environment variables
+    code_entrypoint = os.environ.get("CODE_ENTRYPOINT") or "main.main"
+    code_entrypoint_parts = code_entrypoint.split('.')
+    code_file = code_entrypoint_parts[0]
+    code_func = code_entrypoint_parts[1]
```

Comment:
maybe add some sort of validation for the existence of code_file, code_func and the length of code_entrypoint_parts

Reply:
Very important, must add! Can be cause of exception